### PR TITLE
Layout: Tabs

### LINKED ISSUE #1119: New Layout: Tabs
### Description

Rather than rendering groups side-by-side in a grid or masonry layout (as per Fixed and Grid) and beneath each other (as per Notebook), a new layout option should be available to show content in Tabs.

Vuetify's "Tab" component would assist here: https://vuetifyjs.com/en/components/tabs/#usage

Each group on a given page would be a tab, and would take up the space available on the page.

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Adds a new "Tabs" layout to core Dashboard
- Each group, rendered to a page will instead render to it's own "Tab" on that page.
- Still needs some manual testing

## Related Issue(s)

Closes #1119

## Demo

https://github.com/user-attachments/assets/ee06428f-4c76-483b-a501-ac0f2f263be9

### CODE REVIEW

File: ui/src/layouts/Tabs.vue [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                    >
+                        <v-card variant="outlined" class="bg-group-background">
+                            <template #text>
+                                <widget-group :group="t" :widgets="widgetsByGroup(t.id)" :row-height="rowHeight" />
+                            </template>
+                        </v-card>
+                    </div>
+                </v-tabs-window-item>
+            </v-tabs-window>
+            <!-- <div v-if="orderedGroups" :id="'nrdb-page-' + $route.meta.id" class="nrdb-layout--grid nrdb-ui-page" :class="page?.className">
```
Comment (gayanSandamal) [inline ui/src/layouts/Tabs.vue:35]: Can we do a tree shaking if this is not being used

**Review by gayanSandamal — APPROVED** [review]
Tab view is really cool 👌 Just a comment

You can ignore if it's not necessary.

LGTM

File: ui/src/layouts/Tabs.vue (hunk 1 above)
Comment (joepavitt) [reply ui/src/layouts/Tabs.vue:35]: Yes - absolutely, lazy coding on my part

Comment (gayanSandamal) [reply ui/src/layouts/Tabs.vue:35]: No worries at all 😄

**Comment by omrid01** [comment]
👍👍

**Pull request merged**